larq/optimizers.py

```diff
@@ -1,9 +1,41 @@
+"""Neural networks with extremely low-precision weights and activations, such as 
+Binarized Neural Networks (BNNs), usually contain a mix of low-precision and 
+higher-precision weights (e.g. 8-bit, 16-bit, or 32-bit). Examples of this include the 
+first and last layers of image classificiation models, which have higher-precision 
+weights in most BNN architectures from the literature.
```

Comment:
Should we mention bias or batch norm variables here as well?

Reply:
Discussed this earlier in the AMS office and I don't think so, this is just a simple example of why we need variables (and optimizers) of different precisions at all, and there's no need to explain every use case for that.